### PR TITLE
Release v.1

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Prepare software version 1.0.  After this we can create a GitHub action to create a release and fill in the changelog on merge-to-main event

### Review:
- [x] Code approved by HS

This [version](https://semver.org/) is a:
- [x] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
The test fails because the software version in it is hard coded. A fix is in #28

File: src/schug/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '0.1.0'
+__version__ = "1.0"
```
Comment (henrikstranneheim) [inline src/schug/__init__.py:1]:
```suggestion
__version__ = "1.0.0"
```

**Review by henrikstranneheim — APPROVED** [review]
🎉 👍

**Comment by northwestwitch** [comment]
Before creating the new release, do you think we have to do something on PyPI @henrikstranneheim ?

**Comment by henrikstranneheim** [comment]
We need to check that the tokens exist and that there is a repo at pypi.

**Comment by northwestwitch** [comment]
> We need to check that the tokens exist and that there is a repo at pypi.

When you have time, could you fix it? I'll wait before creating new releases

**Pull request merged**